var/spack/repos/builtin/packages/improved-rdock/package.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class ImprovedRdock(MakefilePackage):
+    """rDock is a fast and versatile Open Source docking program
+    that can be used to dock small molecules against proteins and
+    nucleic acids. """
+
+    homepage = "https://github.com/clinfo/improved_rDock"
+    # url      = "https://sourceforge.net/projects/rdock/files/rDock_2013.1_src.tar.gz"
+    # url = "file://{0}/rDock_2013.1_src_AdvanceSoft_2018.tar.gz".format(os.getcwd())
+    git = "https://github.com/clinfo/improved_rDock.git"
+
+    # version('2013.1', sha256='33eb3aa0c4ede3efe275eb7b7f98c8cb54b0f54d774f400e00cb172e7921b99c')
```

Comment:
We generally prefer to not retain comments like these.  Perhaps modifying the docstring to include the original repository would be appropriate if you want to retain this information.